packages/dataviews/README.md

```diff
@@ -218,6 +218,7 @@ Properties:
 | ----------------------------------------------------------------------------------- | ----- | ---- | ---- |
 | `badgeFields`: a list of field's `id` to render without label and styled as badges. |       | ✓    |      |
 | `styles`: additional `width`, `maxWidth`, `minWidth` styles for each field column.  | ✓     |      |      |
+| `shouldPinActions`: whether to pin actions column when the width is insufficient .  | ✓     |      |      |
```

Comment:
Can this behavior be automatic? cc @jameskoster @matt-west

Reply:
🤔 I don't see a reason why it shouldn't.

Reply:
Would you be open to making it the default behavior and only introduce the flag if there are consumers that actively ask for it?

If we have to do it, my preference would be something that can work for multiple items, rather than individual props for every modifiable behavior. For example, in the past, consumers have shared that they'd want to hide the controls for moving columns around or the config cog, etc.

Reply:
I'm struggling a bit to see scenarios where users or consumers would want to disable this behavior. It feels like a detail to explore separately to me.

Reply:
Ok, let's try to make it the default behavior